test/tap/tests/set_character_set-t.cpp

```diff
@@ -54,7 +54,7 @@ int main(int argc, char** argv) {
 	}
 
 	if (mysql_query(mysql, "set names 'utf8'")) {
-	    fprintf(stderr, "SET CHARACTER SET 'utf8': Error: %s\n",
+	    fprintf(stderr, "SET NAMES 'utf8': Error: %s\n",
```

Comment:
Mixed indentation of tabs with spaces, this is repeated in this particular lines for error reporting across the tests files.

Reply:
Fixed.